test/uk/gov/hmrc/helptosave/connectors/UserDetailsConnectorSpec.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosave.connectors
+
+import java.time.LocalDate
+
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.{Matchers, WordSpec}
+import play.api.libs.json.Json
+import uk.gov.hmrc.helptosave.config.WSHttp
+import uk.gov.hmrc.helptosave.connectors.UserDetailsConnector.UserDetailsResponse
+import uk.gov.hmrc.helptosave.util._
+import uk.gov.hmrc.play.http.{HeaderCarrier, HttpResponse}
+import uk.gov.hmrc.play.test.WithFakeApplication
+
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.concurrent.duration._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+
+class UserDetailsConnectorSpec extends WordSpec with WithFakeApplication with Matchers with MockFactory{
```

Comment:
remove extra empty lines in this. 
Import `TestSupport`  so that we can move all common code like 
```
implicit val hc = HeaderCarrier()
val mockHttp = mock[WSHttp]
``` 
into it